wdl/munge.wdl

```diff
@@ -188,7 +188,6 @@ task sumstat_to_vcf {
 
         with gzip.open(sumstat, 'rt') as f:
             h_idx = {h:i for i,h in enumerate(f.readline().strip().split(delim))}
-            h_idx = defaultdict(lambda: 1e9, h_idx)
```

Comment:
if this is exactly the same sumstat_to_vcf, would be better to seaprate to commons wdl and import the task.

Reply:
Ah, of course that would be wise..